docsbox/docs/views.py

```diff
@@ -111,7 +107,11 @@ def post(self, file_id):
             if mimetype not in app.config["CONVERTABLE_MIMETYPES"]:
                 response = {"status": "corrupted" if mimetype == "Unknown/Corrupted" else "non-convertable", "fileType": mimetype}
             else:
-                task = process_convertion.queue(file_path, set_options(request.headers, mimetype), {"filename": remove_extension(filename), "mimetype": mimetype, "save_in_via": save_in_via})
+                if file_id and is_valid_uuid(file_id):
+                    task = process_convertion_by_id.queue(file_id, set_options(request.headers, mimetype), {"filename": remove_extension(filename), "mimetype": mimetype, "save_in_via": save_in_via})
```

Comment:
Let's keep the v1 api unchanged about this process by id change so that we don't need several different implementations in tasks